### DISCUSSION TITLE
rtsp_transport

### DISCUSSION THREAD

**Discussion by ceewanna** [roflcoopter/viseron#631] [General]
When I started I had viseron reading from cameras. I have just tried reading rtsp from rtsp server. All the ffmpeg setups seemed to fail. I found this ffprobe line.

[2023-07-09 08:22:44] [DEBUG   ] [viseron.components.ffmpeg.stream.ch1_door] - FFprobe command: ffprobe -hide_banner -loglevel error -print_format json -show_error -show_streams rtsp://*****:*****@192.168.1.155:8554/ch1-door

When ffprobe command was tested on command line it gave.
[rtsp @ 0x56550dab6040] method SETUP failed: 461 Unsupported Transport

However, if "-rtsp_transport tcp" was added to the ffprobe command, there was no such error.

In viseron ffmpeg module, there is rtsp_transport select (optional, default: tcp).
Questions:
1. I wonder if ffprobe should also carry such setting by default.
2. I tried also explicit add the option in the config file but the ffprobe command that came out in debug still didn't carry the setting.

**Comment by roflcoopter** [comment]
This was actually solved some time back
